### ISSUE TITLE
Add support for Cities: Skylines

### ISSUE THREAD

**Issue by milkomeda15** [VovoloGames/WorkshopDL#57]
**Provide the game's name below:**
Cities: Skylines

**Steam Storepage**
https://store.steampowered.com/app/255710/Cities_Skylines/

**Additional Context**
Sandbox game about building cities, the default game is a bit weak in content, but with mods, you can almost create realistic cities by adding mods and assets.
Game is full of mods and assets on the workshop, more than Civilization VI, Europa Universalis 4 and RUST combined. I think a lot of people want access to it since it was given in Epic Games and also maybe due to be a single-player game, many people also might have a cracked version. Also popular mods are frenquently updated to fix bugs/errors, to make it compatible with other new mods and to keep it stable with the game updates too, so even I have my favorite mods already downloaded, I would like to update them to prevent bugs and errors when the main game updates his versions.

Thank you.

**Comment by milkomeda15** [comment]
Hello,
I'm sorry, I tried to add the label tag to it but the option was locked for some reason.
I tried downloading using the Nether multiple times, waited 20~30 min each time, tried different workshop urls, but the download doesn't fill the bar and doesn't change anything (empty download files only). I confirmed the numbers and they're all correct (the game ID and the mod ID), ran the WorkshopDL as admin and nothing changed. I thought it was unavailable.
I'm doing anything wrong?

(I paste the workshop mod ID into the url box and the program fills the game ID automatically, I confirm that the game ID is correct, same number on the store page of the game, and of course select the nether option)
